python_scripts/make_merged_CDE_table.py

```diff
@@ -0,0 +1,50 @@
+#!/bin/env python
```

Comment:
@ekopylova I'll be running the QIIME metadata file generation locally. As long as the Python/R CGC API works, building the tables and associated mapping file will be relatively straightforward.